tests/components/landisgyr_heat_meter/test_sensor.py

```diff
@@ -131,15 +154,9 @@ async def test_exception_on_polling(mock_heat_meter, hass: HomeAssistant) -> Non
     assert state.state == STATE_UNAVAILABLE
 
     # Now 'enable' and see if next poll succeeds
-    mock_heat_meter_response = MockHeatMeterResponse(
-        heat_usage_gj=124.0,
-        heat_usage_mwh=None,
-        volume_usage_m3=457.0,
-        heat_previous_year_gj=112.0,
-        heat_previous_year_mwh=None,
-        device_number="devicenr_789",
-        meter_date_time=dt_util.as_utc(datetime.datetime(2022, 5, 19, 20, 41, 17)),
-    )
+    mock_response_gj_changed = mock_response_gj
+    mock_response_gj_changed["heat_usage_gj"] = 124.0
+    mock_heat_meter_response = HeatMeterResponse(**mock_response_gj_changed)
```

Comment:
You would inadvertently change the mock_response_gj constant here.
I suggest instead to "bump" the value in the response - not in the dict.
```suggestion
    mock_heat_meter_response = HeatMeterResponse(**mock_response_gj)
    mock_heat_meter_response.heat_usage_gj += 1
```